README.md

```diff
@@ -17,8 +17,8 @@ Read ["Installation"] from [The Book].
 _Note: If you wish to contribute to the compiler, you should read [this
 chapter][rustcguidebuild] of the rustc-dev-guide instead of this section._
 
-The Rust build system has a Python script called `x.py` to bootstrap building
-the compiler. More information about it may be found by running `./x.py --help`
+The Rust build system has a Python script called `x.py` to bootstrap when building
```

Comment:
I'm not sure about this word "when", it sounds sort of strange to me...

Reply:
True, but "bootstrap building the compiler" doesn't sound quite right either, so what would you suggest in place of that?

Reply:
It doesn't bother me, but I never thought that carefully about it. I'm not sure we even need the word bootstrap in there, we could just say "The rust build system uses a Python script called `x.py` to build the compiler." If we want to, we could add "x.py manages the bootstrapping process."

Reply:
That makes sense, will change it to that!